### PR TITLE
Fix header resolution

### LINKED ISSUE #2: Card headers are different sizes, makes card standardization difficult
Things like this will happens with how we layout our cards https://cdn.discordapp.com/attachments/745465408848527360/878136854724116570/unknown.png

With different sized headers, we end up having a height that's not one size fits all. We can fix this by making the header images all the same resolution.

### PR DESCRIPTION
**Pull request by zapplecat**
Fixes #2

There might be a smarter way to calculate the various larger screen sizes, but I'm not privy to them (actually maybe I can just do addition?). If someone knows a better way let me know.

### CODE REVIEW

File: frontend/src/shared/components/baseCard/baseCard.css [hunk 1]
```diff
@@ -87,21 +87,28 @@
...
     background-position-x: center;
     background-repeat: no-repeat;
 }
 
-@media (max-width: 768px) {
+/*As more columns show up the vw calculation needs to be specifically adjusted*/
+@media (min-width: 1024px) {
     .card-header {
-        height: 25vh;
+        height: 8vw;
```
Comment (DuongTyler) [inline frontend/src/shared/components/baseCard/baseCard.css:99]: Did you mean vh? not vw.

Also I think vh is what is causing all this headache over resizing and stuff. May be better to just set fixed widths and heights for cards at different resolutions and let the tiling take care of the rest

File: frontend/src/shared/components/baseCard/baseCard.css [hunk 2]
```diff
@@ -87,21 +87,28 @@
...
+@media (min-width: 1440px) {
     .card-header {
-        height: 12vh;
+        height: 6vw;
+    }
+}
+
+@media (min-width: 2560px) {
+    .card-header {
+        height: 4vw;
```
Comment (DuongTyler) [inline frontend/src/shared/components/baseCard/baseCard.css:111]: vw or vh? probably better to just change everything to hard px values

**Review by DuongTyler — CHANGES_REQUESTED** [review]

File: frontend/src/shared/components/baseCard/baseCard.css (hunk 1 above)
Comment (zapplecat) [reply frontend/src/shared/components/baseCard/baseCard.css:99]: Originally what's what I also thought, but vw is actually a much better predictor for the number of columns and the height of the cards. If we use vh, and do some calculation, based off the card element size, maybe we can get closer? I'm not sure what the more elegant solution is.

Comment (Lr-Jim) [reply frontend/src/shared/components/baseCard/baseCard.css:99]: yeah, I think having hard px values is better if we have a fixed same for the header and we must take into account the width of the base card also changed based on the resolution. The solution I can think of is
`
.card-header {
  width: 100% 
}
`
Then we can have a forced rule that all headers must match a resolution aspect (4:3 or something)

Comment (DuongTyler) [reply frontend/src/shared/components/baseCard/baseCard.css:99]: I agree with Jim. Enforcing the aspect ratio is probably the best fix here.

**Review by Lr-Jim — APPROVED** [review]

**Review by DuongTyler — APPROVED** [review]

**Pull request merged**